src/Core/src/Platform/Android/PickerExtensions.cs

```diff
@@ -16,6 +16,8 @@ public static void UpdateTitleColor(this MauiPicker platformPicker, IPicker pick
 			{
 				if (PlatformInterop.CreateEditTextColorStateList(platformPicker.TextColors, titleColor.ToPlatform()) is ColorStateList c)
 					platformPicker.SetHintTextColor(c);
+				else if(picker.TextColor == picker.TitleColor)
+					platformPicker.SetHintTextColor(titleColor.ToPlatform());
```

Comment:
Why do we have this color state list instead of just always setting the color?

Reply:
I don't know. Maybe it should be removed... However, I'm not sure of the potential implications removing it might habe